CHANGELOG.md

```diff
@@ -0,0 +1,605 @@
+# Changelog
+
+## 2.5.5
+Dec 06, 2019
+
+-   [JENKINS-39682](https://issues.jenkins-ci.org/browse/JENKINS-39682) -
+    Trigger after-save hooks and recalculate after `createProjectFromXML`
+    or `updateByXml` ([PR\#158](https://github.com/jenkinsci/branch-api-plugin/pull/158))
+-   [PR\#162](https://github.com/jenkinsci/branch-api-plugin/pull/162) -
+    Add CHANGE_BRANCH and CHANGE_FORK descriptions
+-   [PR\#167](https://github.com/jenkinsci/branch-api-plugin/pull/167) -
+    Fix PCT failure on 2.190.x
+-   [JENKINS-59756](https://issues.jenkins-ci.org/browse/JENKINS-59756) -
+    Do not duplicate input in NameEncoder.decode ([PR\#168](https://github.com/jenkinsci/branch-api-plugin/pull/168))
+
+## 2.5.4
+Jul 25, 2019
+
+-   [JENKINS-54864](https://issues.jenkins-ci.org/browse/JENKINS-54864) -
+    Getting issue details... STATUS  Undeprecate the "Automatic branch
```

Comment:
Looks like the plugin was using fancy Jira links on the wiki, so "Getting issue details... STATUS" shows up in a few places here and should be deleted.

```suggestion
    Undeprecate the "Automatic branch
```